src/styles.scss

```diff
@@ -68,6 +68,10 @@ body,
       padding: 10px;
       z-index: 1;
     }
+
+    &-Sort-Icon {
```

Comment:
Please check if this is directly under `.React-Sticky-Table` scope. If not, we should ideally move it there to reduce nesting.